src/components/logo/logo.tsx

```diff
@@ -27,14 +30,57 @@ const mapStateToProps = state => ({
  */
 @connect(mapStateToProps)
 @withPlayer
+@withEventManager
 @withLogger(COMPONENT_NAME)
 @withText({logoText: 'controls.logo'})
 class Logo extends Component<any, any> {
+  /**
+   * @constructor
+   * @param {*} props props
+   */
+  constructor(props: any) {
+    super(props);
+    this.setState({isUrlClickable: true, urlLink: props.config.url});
+  }
+
+  /**
+   * when component did mount
+   *
+   * @returns {void}
+   * @memberof Logo
+   */
+  public componentDidMount(): void {
+    this._handleLogoUrl();
+  }
+
+  /**
+   * handles the logo url
+   * if the url contains ${entryId}, then replace it with the played entry id
+   *
+   * @returns {void}
+   * @memberof Logo
+   */
+  private _handleLogoUrl(): void {
```

Comment:
I think the accessibility Is redundant here
We do not have this in the rest of components

Reply:
I am getting TS errors in my IDE.. it is asking me to add them.. like you did for kaltura-player and playkit-js in TS migration.
if i remove it looks like this-
![image](https://github.com/kaltura/playkit-js-ui/assets/79077248/7ce55c92-f9a9-470d-919a-605dd42ada77)

Reply:
using TS private should have private prefix, public can be ingored. We will add this to our coding standart